src/test/resources/features/Scoring/Question/Question.feature

```diff
@@ -0,0 +1,102 @@
+@Question @Regression
+Feature: Question in Question Bank
+
+  Background:
+    Given user prepare auth request
+    And user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user prepare question bank request
+    And user hit get all question banks endpoint
+    And get first question bank data id
+    And user prepare question request
```

Comment:
I found it best to always call `And user hit logout endpoint` in each `Background`, since previous tests may alter the state of `authData` singleton